nujo/utils/data/dataset.py

```diff
@@ -13,31 +13,35 @@ class Dataset:
 
     Parameters:
     -----------
-    name : will be downloaded from the UCI ML repo (for now)
+    - name : str, will be downloaded from the UCI ML repo (for now)
+    - type : str, type of dataset - either csv or image
+    - override : boolean, for loader, will override file if it exists
+    - download : boolean, should it be downloaded from uci repo
```

Comment:
remove the prefix `-` since the other docs in nujo were written without it, otherwise very nice!

Reply:
But without it the doc string will be on one line

Reply:
When displayed by VSCode?

Reply:
So, @ManiacMaxo should I understand that when the docs don't begin with `-` they are represented as single-lined?